### ISSUE TITLE
Button for extracting messages from a thread in message box.

### ISSUE THREAD

**Issue by FProN** [mainIine/foe-helfer-extension#3146]
A button that extracts maybe all messages since a certain date/message from a single thread in message center would be very helpful for me personally. I created a program to count how many times a word appears in a text file (for guild management purposes), however im completely lost when it comes to web coding so even though i tried to create this feature myself, i could not get it to work at all.

To provide a use-case for this feature i can share my idea which im trying to accomplish. I want to keep a log of people who show up to threatened sectors on gbg map. To do it we currently use FoE-Helper's gbg activity window, copy all the updated(showed in green color, look at attachment) players and paste it into a thread. After that as of now we need to copy paste each message individually to a .txt file, which gets repetetive and tiring, and after that we run my program that counts each players nickname appearance in the text file.
With this feature we wouldn't have to copy paste the messages one by one and it would save us a lot of time.

Im sorry if im asking for too much or a very niche feature.
![abccc](https://github.com/user-attachments/assets/6611205e-0dfe-4dbf-abe0-847257bcc69f)

**Comment by Beelzebob6666** [comment]
why don't you copy it directly into the txt file instead of going through the messages?

**Comment by FProN** [comment]
I could and would do that if i was the only one who kept track of the data. However that would mean i'd have to sit in front of gbg map 24/7, because you have to be there when the battle(a race between 2+ guilds) for a sector happenes and refresh the FoE-Helper's gbg activity overview window before the battle for a sector so it doesn't show old guild member's activity.

We currently work with 7 guildmates to write down the data at different times in the day, and we thought that pasting it into the ingame mailbox is the most convenient way to do it. 
Another question that may arise is why wouldn't we copy it into some kind of communicator like discord, but the answear is again, convenience. Not every guild member has/wants to use a communicator outside of the game itself, especially if it requires extra time/steps/clicks, as some of them only clock in for the race from their office job's PC where a browser extension may slide, but a communicator app would be very shady to explain to the boss.

tl;dr: I'm not the only person copying it and it's easier for other people to paste it into ingame mailbox then anywhere else.

**Comment by Beelzebob6666** [comment]
just do a shared google doc then...
